apps/src/code-studio/pd/teacher_application/SummerWorkshopSchedule.jsx

```diff
@@ -1,9 +1,112 @@
 import React from 'react';
+import _ from 'lodash';
 
-export default function SummerWorkshopSchedule() {
-  return (
-    <h2>
-      This section is huge. Putting it off for now.
-    </h2>
-  );
-}
+const group2OrGroup1CsdWorkshops = {
```

Comment:
nit: this looks like `group 20 r Group 1...` Do we need the groups in the variable name? Can it just be `csdWorkshops`? Or maybe something like `groups1or2CsdWorkshops`?